src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala

```diff
@@ -199,14 +207,35 @@ class UserService(
         // validate all fields to be updated
         var errorReports = Seq[ErrorReport]()
         request.email.foreach(email => errorReports = errorReports ++ validateEmail(email, blockedEmailDomains))
+        request.azureB2CId.foreach(azureB2CId => errorReports = errorReports ++ validateAzureB2CId(azureB2CId))
         if (errorReports.nonEmpty) {
           IO.raiseError(new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.BadRequest, "invalid user update", errorReports)))
+        } else if (request.googleSubjectId.contains(GoogleSubjectId("")) && request.azureB2CId.contains(AzureB2CId(""))) {
+          IO.raiseError(
+            new WorkbenchExceptionWithErrorReport(
+              ErrorReport(StatusCodes.BadRequest, "unable to null both azureB2CId and googleSubjectId in the same request", errorReports)
```

Comment:
If I accidentally set both the `azureB2CId` and `googleSubjectId` to null in different requests, would everything break? 😅 Should SAM check/prevent this? Or is it okay because I could just call this endpoint again to fix it?

Reply:
Oh nevermind! I see there's an error that gets thrown when updating the database